kerberos/defaults.yaml

```diff
@@ -6,16 +6,14 @@ kerberos:
     libdefaults:
       default_realm: EXAMPLE.COM
       default_domain: example.com
-      dns_lookup_kdc: false
-      dns_lookup_realm: yes
-      ticket_lifetime: 25h 
+      dns_lookup_kdc: true
```

Comment:
Is it wise to change these defaults on people?

I would like to have someone who actually uses this to review it and say that it is fine to change these defaults.

It seems more reasonable to leave them disabled, and have people enable them if they actually want them.

Daniel

Reply:
hi @gtmanfred  good comment I have cleaned up the defaults.yaml so its clear defaults are software defaults - I checked krb5.conf on Hedimal and MIT and software defaults are largely good IMHO.  I removed dns_lookup_kdc, dns_lookup_realm, overrides from pillar.example because (again) the software defaults are probably good.